### PR TITLE
Updated readme_de.md

### PR DESCRIPTION
**Pull request by ollo-der-echte**
There were some spelling issues, but probably it's all right as it was last version, what do you think?

### CODE REVIEW

File: docs/readme_de.md [hunk 1]
```diff
@@ -170,12 +170,12 @@ Du kannst mehrere, mit Kommas separierte, Werte in der bg_color Option angeben,
...
 - `hide_title` - _(Boolean)_
 - `line_height` - Legt die Zeilenhöhe des Texts fest _(Zahl)_
-- `hide_progress` - Verbirgt die Fortschrittanzeige und Prozentzahl _(Boolean)_
+- `hide_progress` - Verbirgt die Fortschrittsanzeige und Prozentzahl _(Boolean)_
 - `custom_title` - Legt einen benutzerdefinierten Titel fest
 - `layout` - Wechselt zwischen zwei verschiedenen Layouts: `default` & `compact`
 - `langs_count` - Begrenzt die Anzahl der angezeigten Sprachen auf der Karte
 - `api_domain` - Legt eine benutzerdefinierte API Domain fest, z.B. für [Hakatime](https://github.com/mujx/hakatime) oder [Wakapi](https://github.com/muety/wakapi)
-- `range` – Fragt eine eine Zeitspanne an, als die standardmäßig in WakaTime hinterlegte, z.B. `last_7_days`. Siehe [WakaTime API Dokumentation](https://wakatime.com/developers#stats).
+- `range` – Fragt eine Zeitspanne an, als die standardmäßig in WakaTime hinterlegte, z.B. `last_7_days`. Siehe [WakaTime API Dokumentation](https://wakatime.com/developers#stats).
```
Comment (Morphclue) [inline docs/readme_de.md:178]:
```suggestion
- `range` – Fragt eine andere Zeitspanne an, als jene, welche standardmäßig in Wakatime hinterlegt ist. Zum Beispiel `last_7_days`. Siehe [WakaTime API Dokumentation](https://wakatime.com/developers#stats).
```

File: docs/readme_de.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ Hinweis: Seit [#58](https://github.com/anuraghazra/github-readme-stats/pull/58)
    ![](https://files.catbox.moe/pqub9q.png)
 1. Wähle root und füge eine Umgebungsvariable namens PAT_1 (siehe Abbildung) die als Wert deinen persönlichen Access Token (PAT) hat hinzu, den du einfach [hier](https://github.com/settings/tokens/new) erzeugen kannst (lasse alles wie es ist, vergebe einen beliebigen Namen)
    ![](https://files.catbox.moe/0ez4g7.png)
-1. Klicke deploy, und das wars. Besuche deine Domains um die API zu benutzen!
+1. Klicke auf "Deploy", und das wars. Besuche deine Domains um die API zu benutzen!
```
Comment (Morphclue) [inline docs/readme_de.md:352]:
```suggestion
1. Klicke auf `Deploy`, und das wars. Besuche deine Domains um die API zu benutzen!
```

**Review by Morphclue — CHANGES_REQUESTED** [review]
I think these changes would be benefitial.

File: docs/readme_de.md (hunk 2 above)
Comment (ollo-der-echte) [reply docs/readme_de.md:352]: Yea, that's way better, I'm sorry

**Review by w3bdesign — APPROVED** [review]

**Review by Morphclue — APPROVED** [review]

**Comment by rickstaa** [comment]
@Morphclue thanks for reviewing this PR. Now it has the two reviews required for merging translation-related PRs. I merged it into the main branch.

**Pull request merged**